volume-group-operator/config/crd/bases/csi.ibm.com_volumegroupcontents.yaml

```diff
@@ -46,10 +46,10 @@ spec:
           description: VolumeGroupContent is the Schema for the volumegroupcontents API
           properties:
             apiVersion:
-              description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+              description: "APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources"
```

Comment:
why the `"`,`'` diff?

Reply:
this is how the Yaml generation does it I think your IDE changed it